docs/development/workflow/git_edit_workflow_examples.rst

```diff
@@ -59,7 +59,7 @@ Set up an isolated workspace
   The example below shows the necessary steps in the Miniconda/Anaconda Python
   distribution::
 
-    conda create --name apy-1761 python=3.9 # replace 3.9 with desired version
+    conda create --name apy-1761 python=3.10 # replace 3.10 with desired version
```

Comment:
If there are any other changes requested, might as well change this to 3.12 so we don't have to repeat it for a while...

Reply:
✅